project/Dependencies.scala

```diff
@@ -5,32 +5,32 @@ object Dependencies {
   object V {
     val bouncyCastle = "1.70"
     val betterFiles = "3.9.1"
-    val cats = "2.7.0"
-    val catsEffect = "3.3.5"
+    val cats = "2.8.0"
+    val catsEffect = "3.3.14"
     val catsRetry = "2.1.0" // not published for CE3 yet
     val circe = "0.14.2"
-    val ciris = "2.3.2"
+    val ciris = "2.3.3"
     val comcast = "3.1.3"
-    val decline = "2.2.0"
+    val decline = "2.3.0"
     val derevo = "0.13.0"
     val doobie = "1.0.0-RC1"
     val droste = "0.9.0"
     val enumeratum = "1.7.0"
-    val h2 = "2.1.212"
+    val h2 = "2.1.214"
     val javaxCrypto = "1.0.1"
-    val flyway = "8.5.12"
-    val fs2 = "3.2.7"
-    val fs2Data = "1.3.1"
+    val flyway = "9.1.0"
```

Comment:
It is a major update. Have you tried backward compatibility of migrations?

Reply:
I guess if the cluster gets up from the scratch and works correctly then the migrations must work as well right?